sdk/arm/keyvault/2019-09-01/armkeyvault/zz_generated_connection.go

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package armkeyvault
+
+const telemetryInfo = "azsdk-go-armkeyvault/<version>"
```

Comment:
Be sure to update the version to `v0.1.0`

Reply:
Fixed